README.md

```diff
@@ -151,3 +154,124 @@ Before PR, please run complete code check via command:
 composer cs:install # only the first time
 composer fix # otherwise pre-commit hook can fail
 ````
+
+## Math Framework tools structure
+
+The biggest advantage is that you can choose which layer best fits
+your needs and start build on the top of it, immediately, without the need
+to create everything by yourself. Our tools are tested for bugs
+and tuned for performance, so you can save a significant amount
+of your time, money, and effort.
+
+We tend to be modular as much as possible, so you should be able
+to create an extension on each layer and its sublayers.
+
+**Mathematical framework layers** ordered from the most concrete
+one to most abstract one:
+
+<table>
+    <tr>
+        <td>
+            <b>
+            <a href="https://github.com/cothema/math-php-api">
```

Comment:
Why is this package first?

Reply:
Because it is the most concrete one (as said in the text). Please suggest a change.

Reply:
Or should I change the order from the most abstract one to the most concrete one?

Reply:
I think the Mathematicator should only use the packages it guarantees.

If it is a third-party package, it must be marked.

Reply:
Ok, I'll move this one to the end and mark it as a third party package.